pkg/client/load.go

```diff
@@ -102,6 +102,7 @@ func (c *Cassowary) runLoadTest(outPutChan chan<- durationMetrics, workerChan ch
 		}
 
 		request = request.WithContext(httptrace.WithClientTrace(context.Background(), trace))
+		c.Client.Transport.(*http.Transport).Proxy = http.ProxyFromEnvironment
```

Comment:
could you please move this logic to where c.Client is initialized in the Coordinate method

Reply:
and thanks for the contribution, I apologise for late reply

Reply:
Done.